docs/langref/src/concepts/layouting.md

```diff
@@ -67,74 +67,73 @@ Additionally, the `width` and `height` properties can also be specified as a `%`
 unit, which applies relative to the parent element. For example a `width: 50%` means half
 of the parent's `width`.
 
-The default values for `x` and `y` properties are set such that elements are centered within their
+The default values for `x` and `y` properties are such that elements are centered within their
 parent.
-(Unless using the legacy syntax, in which case the default is `0, 0`)
 
-The default values for `width` and `height` depend on the type of element. Some elements are sized
+The default values for `width` and `height` depend on the element. Some elements are sized
 automatically based on their content, such as `Image`, `Text`, and most widgets. The following elements
-do not have content and therefore default to fill their parent element:
+don't have content and default to fill their parent element:
 
-* `Rectangle`
-* `TouchArea`
-* `FocusScope`
-* `Flickable`
+-   `Rectangle`
+-   `TouchArea`
+-   `FocusScope`
+-   `Flickable`
 
-## Automatic Placement using Layouts
+## Automatic Placement Using Layouts
 
 Slint comes with different layout elements that automatically calculate the position and size of their children:
 
-* `VerticalLayout` / `HorizontalLayout`: The children are placed along the vertical or horizontal axis.
-* `GridLayout`: The children are placed in a grid of columns and rows.
+-   `VerticalLayout` / `HorizontalLayout`: The children are placed along the vertical or horizontal axis.
+-   `GridLayout`: The children are placed in a grid of columns and rows.
 
-Layouts can also be nested, making it possible to create complex user interfaces.
+You can also nest layouts, making it possible to create complex user interfaces.
 
 You can tune the automatic placement using different constraints, to accommodate the design of your user
 interface. For example each element has a minimum and a maximum size. Set these explicitly using the
 following properties:
 
-* `min-width`
-* `min-height`
-* `max-width`
-* `max-height`
+-   `min-width`
+-   `min-height`
+-   `max-width`
+-   `max-height`
 
 A layout element also affects the minimum and maximum size of its parent.
 
-An element is considered to have a fixed size in a layout when the `width` and `height` is specified directly.
+Any element with a specified `width` and `height` has a fixed size in a layout.
 
 When there is extra space in a layout, elements can stretch along the layout axis. You can control this stretch
 factor between the element and its siblings with these properties:
 
-* `horizontal-stretch`
-* `vertical-stretch`
+-   `horizontal-stretch`
+-   `vertical-stretch`
 
-A value of `0` means that the element will not be stretched at all; unless all siblings also have a stretch
-factor of `0`. Then all the elements will be equally stretched.
+A value of `0` means that the element won't stretch at all, unless all siblings
+also have a stretch factor of `0`. In this case all elements stretch equally.
```

Comment:
It's still a little unclear if "In this case" refers to "A value of `0`" or "all siblings have a stretch factor of `0`". How about this?

```suggestion
A value of `0` means that the element won't stretch at all. All elements stretch equally if they all have a stretch factor of `0`.
```